src/common/interfaces/loggerJson.interface.ts

```diff
@@ -0,0 +1,27 @@
+import { Visibility } from 'src/common/enums/visibility.enum';
+import { ReqBody } from 'src/endpoints/interface/endpoint.interface';
+import { HttpMethod } from '../enums/httpMethods.enum';
+
+
+export type LoggerJson = {
```

Comment:
@SunmadeBabafemi hope these required values wont throw an error sometime in d future

Reply:
not at all.. there are not required perse... they are fields that the frontend can use to populate changed fields